### DISCUSSION TITLE
documentation: windows binaries, what about the static build?

### DISCUSSION THREAD

**Discussion by massimosala** [libvips/libvips#2937] [Q&A]
General question: I don't found anything about the *static* build in the docs.
Pls clarify.

Specific question: vips with PHP 7.4 (thread safe). Should I get
`vips-dev-w64-web-8.12.2.zip`
or
`vips-dev-w64-web-8.12.2-static.zip`

Thanks, Sala

**Comment by massimosala** [comment]
Sorry for the bug label, I cannot found how to change it.

**Comment by jcupitt** [comment]
I think either is fine. The static one might be slightly smaller and faster.
